### DISCUSSION TITLE
Load players loadout on respawn

### DISCUSSION THREAD

**Discussion by TheLocalPub** [Vdauphin/HeartsAndMinds#1603] [Question and answer] [answered]
Currently it would appear that if a user dies, they'll respawn with whatever loadout of the placed unit in the editor is.
Is there a way for clients to respawn with what they last had or a pre-defined loadout?

**Comment by Vdauphin** [comment]
Hello,

> Is there a way for clients to respawn with what they last had 

You have an option in ACE: https://ace3.acemod.org/wiki/feature/respawn

> or a pre-defined loadout?

See https://community.bistudio.com/wiki/Arma_3:_Respawn

Also by default, H&M allow player to access to ACE3 Arsenal what ever the respawn location the player selected

Cheers

**Comment by TheLocalPub** [comment, marked as answer]
Sorted.

The method used to make clients respawn with the original loadout was to:
1. In the editor, go to the attributes tab at the top, go into multiplayer tab.
2. In the "respawn" collapsible menu, tick "Save Loadout" at the bottom.

Players will now respawn with their original loadout.
